src/Mod/Draft/draftmake/make_patharray.py

```diff
@@ -104,8 +106,9 @@ def make_path_array(baseobject,pathobject,count,xlate=None,align=False,pathobjsu
         else:
             ViewProviderDraftArray(obj.ViewObject)
             gui_utils.formatObject(obj,obj.Base)
-            if len(obj.Base.ViewObject.DiffuseColor) > 1:
-                obj.ViewObject.Proxy.resetColors(obj.ViewObject)
+            if hasattr(obj.Base.ViewObject, "DiffuseColor"):
+                if len(obj.Base.ViewObject.DiffuseColor) > 1:
+                    obj.ViewObject.Proxy.resetColors(obj.ViewObject)
```

Comment:
I'm working on a branch where I improve some things on the arrays, and I also noticed this. I also guarded it.

Reply:
If  you are working on arrays, is this PR going to cause problems?

Reply:
You can merge this, and I will fix the conflicts afterwards.